### ISSUE TITLE
Lost all my local data

### ISSUE THREAD

**Issue by diawl** [a85/POSTMan-Chrome-Extension#7431]
I don't know what happened behind the scene. I'm always using Mac native app.

But today I just lost all my local data including environment setting/api collections/scripts I have built over the last 5 months. 

I don't understand why or how this would happen. Just depressed and disappointed.

**Comment by Ghusar** [comment]
@diawl can you write to us help@getpostman.com so that we take a closer look.

**Comment by Ghusar** [comment]
@diawl Meanwhile can you open https://go.postman.co/me/collections and see if you can find your collections there?

**Comment by diawl** [comment]
@Ghusar No collections in sync. I probably didn't even login. It worked without logging in until everything was gone. 😂

**Comment by preethammavin** [comment]
@diawl I would highly recommend keeping the collections in sync so that it is backed up continuously or keep data backed up locally. 
I would like to know did you happen to do any action for this to occur, was it an update or any other action?

**Comment by diawl** [comment]
Now I have learned it the hard way, the sync is on. 

I was trying to open it then the postman window just didn't show up(another issue I often see), I had to right click the app on dock then click `new window`. But this time it didn't work either. 

Then I just tried to kill the app and open it again. This time I realised two Postman apps on my laptop, one in `~/Application` one in `~/Download`. I tried to open the one in `~/Download` and it asked me to put it in `Application` folder, I accidentally agreed. Then it opened, but with everything empty. 

This is what I can remember. 

I just felt frustrated and wanted to ask: if user can easily lose their local data, why allow them to start without login in the first place?
